micromanager_gui/main_window.py

```diff
@@ -114,7 +115,19 @@ def __init__(self, viewer: napari.viewer.Viewer, remote=True):
         self.objectives_cfg = None
 
         # create connection to mmcore server or process-local variant
-        self._mmc = RemoteMMCore() if remote else CMMCorePlus()
+        _v = bool(os.getenv("MM_VERBOSE"))
+        self._mmc = RemoteMMCore(verbose=_v) if remote else CMMCorePlus()
```

Comment:
This is maybe pre-mature? without any changes to threading the default will freeze up the UI. 

Also should document this in the readme.

Reply:
yeah, can do in another PR.  however, I _think_ most of the things we actually need to be threaded are within pymmcore (not here)

Reply:
My understanding of device drivers is that modern ones (and some of the older ones) spawn threads and return immediately from functions, with the notable exception of `snap_image`